services/cron/tasks_extended.go

```diff
@@ -99,6 +99,16 @@ func registerRepositoryUpdateHook() {
 	})
 }
 
+func registerRepositoryUpdateConfig() {
+	RegisterTaskFatal("resync_all_configs", &BaseConfig{
+		Enabled:    false,
+		RunAtStart: false,
```

Comment:
Seems as though maybe this should run at start, however the same can be said of hooks, so I left it be.